extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -895,6 +895,32 @@ describes.repeated(
             expect(element.getAttribute('src')).to.equal('');
           });
 
+          it('should not render if [src] has changed since the fetch was initiated', async () => {
```

Comment:
Does this test fail if you revert the changes in amp-list.js?

Reply:
Yep!